compose/compose.go

```diff
@@ -648,16 +678,66 @@ func main() {
 		services[svc.name] = svc
 	}
 
+	// Alpha Customization
+	customAlphas := make(map[int]string)
+	if len(opts.CustomAlphaOptions) > 0 {
```

Comment:
You don't need this length check. If the list is empty, the for loop will not be executed by itself.